server/methods/core/orders.js

```diff
@@ -9,18 +9,36 @@ import { SSR } from "meteor/meteorhacks:ssr";
 import { Media, Orders, Products, Shops, Packages } from "/lib/collections";
 import { Logger, Hooks, Reaction } from "/server/api";
 
+/**
+ * @file Methods for Orders.
+ *
+ *
+ * @namespace Methods/Orders
+*/
 
-// helper to return the order credit object
-// credit paymentMethod on the order as per current active shop
-// returns entire payment method
+/**
+ * @name orderCreditMethod
+ * @summary Helper to return the order credit object.
```

Comment:
Does this one need a `memberof Methods/Orders` tag?

Reply:
✔️